src/assets/javascript/analytics/ga4/pageViewTracker.js

```diff
@@ -0,0 +1,54 @@
+/* global window document */
+
+window.DI = window.DI || {}
+window.DI.analyticsGa4 = window.DI.analyticsGa4 || {}
+window.DI.analyticsGa4.trackers = window.DI.analyticsGa4.trackers || {};
+
+(function (trackers) {
+
+  'use strict'
+
+  let PageViewTracker = {
+
+    init: function() {
+      const data = {
+        event: 'page_view_ga4',
+        page_view: {
+          language: this.getLanguage(),
+          location: this.getLocation(),
+          organisations: '<OT1056>',
+          primary_publishing_organisation: 'government digital service - digital identity',
+          status_code: this.getStatusCode(),
+          title: this.getTitle(),
+          referrer: this.getReferrer(),
+          taxonomy_level1: 'web cri',
+          taxonomy_level2: 'cic'
+        }
+      }
+      window.DI.core.sendData(data)
+    },
+
+    getLanguage: function() {
+      return (window.DI.cookies.getCookie('lng') ?? 'en').toLowerCase()
+    },
+
+    getStatusCode: function() {
+      return window.DI.httpStatusCode ?? 200
```

Comment:
I can't see `window.DI.httpStatusCode` being set in the code base, where are we expecting that this might come from?

Reply:
This is a good point, it was done this way in the original PR by DCMAW because they set error codes on their error pages e.g. https://github.com/alphagov/di-ipv-dca-front/blob/3b526cf82f4eab21795e99c3475ba1ccb9789cbe/src/views/errors/error.njk#L36
I don't think we are actually responsible for the display of error pages but the ask is for the value to be dynamic (https://govukverify.atlassian.net/wiki/spaces/PI/pages/3603791886/WIP+GA4+Implementation+Guide#On-page-load-IN-SCOPE:~:text=Alphanumeric-,status_code,-HTTP%20response%20status), open to suggestions if this needs to be changed

Reply:
Sorry the github link above is showing not found, I'm guessing this might be something to do with migrations but the new ipv-dca-front is empty https://github.com/govuk-one-login/ipv-dca-frontend. Do you happen to have the new link or code snippet I can see?

Reply:
Ah sorry that's my bad I forgot you need permission to access that repo - you can join this team https://github.com/orgs/alphagov/teams/di-frontend-developers which will give you access, alternatively this is what I was trying to show you 
![image](https://github.com/alphagov/di-ipv-cri-cic-front/assets/118540036/da9b5f80-df74-44c2-a488-17d3b8dee3c7)
Snippet above is from an error page

Reply:
Theoretically we could on our error pages however we don't currently use them as we do callback() when we get an error to redirect back to RP

Reply:
Right ok so I guess it would only be if the redirect to the RP failed but we use the error page from common express so we can't change that anyway?